.editorconfig

```diff
@@ -1,2 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
There are a couple of places where we use trailing whitespace and removing it would break the code. I'm not generally opposed to this. But it should only be added after the code has been changed. Until then, please remove this line.

Reply:
ed578a6c6348fb9ff7e9cc78a2a771b5f06b3f89